src/middlewared/middlewared/plugins/system.py

```diff
@@ -524,27 +524,39 @@ async def dmidecode_info(self):
             self.DMIDECODE_CACHE['ecc-memory'] = ecc
 
         if self.DMIDECODE_CACHE['baseboard-manufacturer'] is None:
-            bm = ((await run(["dmidecode", "-s", "baseboard-manufacturer"], check=False)).stdout.decode(errors='ignore')).strip()
+            bm = (
+                (await run(["dmidecode", "-s", "baseboard-manufacturer"], check=False)).stdout.decode(errors='ignore')
```

Comment:
Can we please be consistent with quotes ? Like use single or double but not both ?

Reply:
That's how it was before but I agree, I will go back and fix it.